src/partition.rs

```diff
@@ -857,7 +857,8 @@ pub fn get_intra_edges<T: Pixel>(
       let dc_or_cfl =
         mode == PredictionMode::DC_PRED || mode == PredictionMode::UV_CFL_PRED;
 
-      needs_left = mode != PredictionMode::V_PRED && (!dc_or_cfl || x != 0);
+      needs_left = mode != PredictionMode::V_PRED && (!dc_or_cfl || x != 0)
+        && !(mode == PredictionMode::D45_PRED || mode == PredictionMode::D63_PRED);
```

Comment:
This was an unrelated fix right? Just a speedup?

Reply:
You are right, not related and just wanted to remove unused computations.